ext/node/update_node_stream.js

```diff
@@ -0,0 +1,507 @@
+#!/usr/bin/env -S deno run --allow-read --allow-write --allow-env --allow-run
+// deno-lint-ignore-file
+// Copyright 2018-2025 the Deno authors. MIT license.
+
+// This file is used to transform Node.js internal streams code to
+// Deno polyfills.
```

Comment:
Can you expand this description? This file is only run when we want to upgrade the node streams code? 

Can you use TypeScript instead?

Reply:
Ported to TypeScript